vip-helpers/vip-utils.php

```diff
@@ -990,11 +990,31 @@ function wpcom_vip_load_plugin( $plugin = false, $folder = false, $load_release_
 	// Array of directories to check for the above files in, in priority order
 	$test_directories = array();
 
+	// Array of shared plugins we are not deprecating
+	$protected_shared_plugins = array(
+		'two-factor',
+		'jetpack-force-2fa',
+	);
+
+	// Array of shared plugins that we cannot move
+	$retired_shared_plugins = array(
+		'vip-go-elasticsearch',
+	);
+
 	if ( $folder ) {
 		$test_directories[] = WP_PLUGIN_DIR . '/' . $folder;
 	} else {
 		$test_directories[] = WP_PLUGIN_DIR;
-		$test_directories[] = WPMU_PLUGIN_DIR . '/shared-plugins';
+		if ( defined( 'WPCOM_VIP_DISABLE_SHARED_PLUGINS' ) ) {
```

Comment:
We should also check that the value of the constant equals `true`, so that `define( 'WPCOM_VIP_DISABLE_SHARED_PLUGINS', false );` doesn't trigger it.